src/features/Dashboard/Dashboard.component.tsx

```diff
@@ -0,0 +1,61 @@
+import { useAppThunkDispatch, useAppSelector } from "hooks";
+import React, { useEffect } from "react";
+import { Container, Row, Col } from "react-bootstrap";
+import { useTranslation } from "react-i18next";
+import * as actions from "./Dashboard.actions";
+
+import { MoviesList } from "./MoviesList";
+
+export const Dashboard = () => {
+  const { t } = useTranslation();
+  const dispatch = useAppThunkDispatch();
+  const activeLanguage = useAppSelector((state) => state.language.activeLanguage);
+  const {
+    isLoadingNowPlayingMovies,
+    nowPlayingMovies,
+    isLoadingUpcomingMovies,
+    upcomingMovies,
+    isLodaingPopularMovies,
+    popularMovies,
+  } = useAppSelector((state) => state.dashboard);
+  const getNowPlayingMoviesAvailable = !nowPlayingMovies.length && !isLoadingNowPlayingMovies;
```

Comment:
To my mind it's strange name, I do not understand what it means

Reply:
Renamed to nowPlayingMoviesRequestAvailable, upcomingMoviesRequestAvailable, popularMoviesRequestAvailable